src/platforms/ctxlink/platform.h

```diff
@@ -105,6 +105,12 @@
 #define USB_DP_PIN  GPIO12
 #define USB_DM_PIN  GPIO11
 
+/*
+ * To use USART1 as USBUSART, DMA2 is selected from RM0368, page 170, table 29.
+ * This table defines USART1_TX as stream 7, channel 4, and USART1_RX as stream 2, channel 4.
+ * Because USART1 is on APB2 with max Pclk of 100 MHz,
+ * reachable baudrates are up to 12.5M with OVER8 or 6.25M with default OVER16 (per DS10314, page 31, table 6)
```

Comment:
Please reduce to 84 MHz for F401, giving 10.5M and 5.25M. The datasheet number is also different for this part.

Reply:
Thanks for the heads-up ... fixed